src/controllers/TransactionsTableController/TransactionsTableController.ts

```diff
@@ -39,6 +39,14 @@ export class TransactionsTableController {
       })
     );
 
+    const getExplorerUrl = (relativePath?: string) => {
```

Comment:
it's strange to have an optional argument that makes the function return undefined if not provided